addons/sale_management/controllers/portal.py

```diff
@@ -54,6 +54,7 @@ def update_line_dict(self, line_id, remove=False, unlink=False, order_id=None, a
             'order_amount_total': format_price(order_sudo.amount_total),
             'order_amount_untaxed': format_price(order_sudo.amount_untaxed),
             'order_amount_tax': format_price(order_sudo.amount_tax),
+            'order_tax_table': request.env['ir.ui.view'].render_template('sale.portal_content_tax_table', {'sale_order': order_sudo}),
```

Comment:
You need to check if the new view exists, because it will not most of the time and this will crash (because no upgrade by default).